internal/server/users/get_user/get_user_tests/get_user_test.go

```diff
@@ -0,0 +1,146 @@
+package get_user_tests
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/ShlykovPavel/users-microservice/internal/lib/api/models/users/create_user"
+	"github.com/ShlykovPavel/users-microservice/internal/lib/api/models/users/get_user_by_id"
+	"github.com/ShlykovPavel/users-microservice/internal/lib/api/response"
+	"github.com/ShlykovPavel/users-microservice/internal/server/users/get_user"
+	"github.com/ShlykovPavel/users-microservice/internal/storage/database/repositories/users_db"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"log/slog"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+type MockUserRepository struct {
+	mock.Mock
+}
+
+func (m *MockUserRepository) CreateUser(ctx context.Context, userinfo *create_user.UserCreate) (int64, error) {
+	args := m.Called(ctx, userinfo)
+	return args.Get(0).(int64), args.Error(1)
+}
+
+func (m *MockUserRepository) GetUser(ctx context.Context, userId int64) (users_db.UserInfo, error) {
+	args := m.Called(ctx, userId)
+	return args.Get(0).(users_db.UserInfo), args.Error(1)
+}
+
+func (m *MockUserRepository) CheckAdminInDB(ctx context.Context) (users_db.UserInfo, error) {
+	args := m.Called(ctx)
+	return args.Get(0).(users_db.UserInfo), args.Error(1)
+}
+
+func (m *MockUserRepository) AddFirstAdmin(ctx context.Context, passwordHash string) error {
+	args := m.Called(ctx, passwordHash)
+	return args.Error(0)
+}
+
+func TestGetUser(t *testing.T) {
+	tests := []struct {
+		name           string
+		userId         string
+		setupMock      func(*MockUserRepository)
+		expectedStatus int
+		expectedBody   interface{}
+	}{
+		{
+			name:   "success get user",
+			userId: "1",
```

Comment:
```suggestion
			userId: 1,
```
если есть постфиксы, то 1L